### ISSUE TITLE
Links to cells inside notebook to work in Colab as they do in blob

### ISSUE THREAD

**Issue by uwesterr** [fastai/fastpages#182]
**Is your feature request related to a problem? Please describe.**
If a link within a notebook is created with 
Cell a:
(#uplaod_local_files) (Link only works in blog not in Colab)
Cell b:
<a id='uplaod_local_files'></a>
that works well in the blog, but not once you open in in colab
**Describe the solution you'd like**
it would be great if the links would work in colab as in the blog
**Describe alternatives you've considered**
right now i just write "(Link only works in blog not in Colab)"
**Additional context**
please find example file attached, one none working link is in the line 
"Next step is to train your model in section Upload local files (Link only works in blog not in Colab)"
[2020-03-01-TrainDonkeyCar.ipynb.zip](https://github.com/fastai/fastpages/files/4297459/2020-03-01-TrainDonkeyCar.ipynb.zip)

**Comment by hamelsmu** [comment]
@uwesterr just to make sure I understand, I have some follow up questions:

- What is `#uplaod_local_files` ?  What does that comment do?  I have never seen that before and I cannot seem to find that in the nbdev documentation?  
- Just to verify, is this a case of functionality working in the blog but NOT in Colab?  If so, I'm not sure how to enable functionality in Colab ( as I don't control Colab ), but I can certainly look into it!  

Thank you

**Comment by uwesterr** [comment]
hi @hamelsmu ,
the `#uplaod_local_files` is a label i set in a text cell
`
Step 3: Upload local files
  <a id='uplaod_local_files'></a>
  You can upload files from local machine as well, but probably is slower than above approach   downloading files from Google Drive
`
so i can set a link in another cell 
`
and yes, it works in the blog but not in Colab.
thanks for looking into this issue and have a great weekend
uwe

**Comment by hamelsmu** [comment]
Hello @uwesterr thank you so much for showing your notebook.  I noticed that the syntax highlighting is not that great in your notebook, so I corrected that with #183 in the meantime.  I would suggest upgrading to the latest version of fastpages so this issue can be corrected (even though you never complained about it, I noticed the problem)

Back to your question:

It appears that Colab doesn't seem to support inline links.  I tried it with the normal way it is supported on GitHub etc and it doesn't work https://colab.research.google.com/drive/1rZst_VIxb-mxcWgupj4PJxfHzqszFFKP

Closing this issue for now, but feel free to ask me questions anytime
